#134/Cpp/Solution.cpp

```diff
@@ -0,0 +1,21 @@
+class Solution {
```

Comment:
As per "Step 2: Folder structure and conventions" in contribution guidelines, please address #4.

Reply:
Yeah Sure !!!

Reply:
LGTM